### PR TITLE
[WIP] Move ose tests to sjb

### PR DESCRIPTION
**Pull request by paradigm**
/cc @stevekuznetsov 

This PR is paired with [this PR in ci-config](https://github.com/openshift/ci-config/pull/25).

Double check the test names and categories are desirable.

### CODE REVIEW

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -269,14 +269,50 @@ presubmits:
     trigger: "((?m)^/test service-catalog,?(\\s+|$))"
 
   openshift/ose:
-  - name: test_pull_requests_ose
+  - name: test_pull_request_ose_check_and_verify_tasks
     agent: jenkins
     labels:
       master: ci.dev.openshift.redhat.com
     always_run: true
     context: ci/openshift-jenkins/ose
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]:
```yaml
context: ci/openshift-jenkins/check
rerun_command: "/test check"
trigger: "((?m)^/test( all| check),?(\\s+|$))"
```

File: cluster/ci/config/prow/config.yaml [hunk 2]
```diff
@@ -269,14 +269,50 @@ presubmits:
...
-    trigger: "((?m)^/test( all| ose),?(\\s+|$))"
+    rerun_command: "/test check_and_verify_tasks"
+    trigger: "((?m)^/test( all| check_and_verify_tasks),?(\\s+|$))"
+
+  - name: test_pull_request_ose_conformance
+    agent: jenkins
+    labels:
+      master: ci.dev.openshift.redhat.com
+    always_run: true
+    context: ci/openshift-jenkins/ose
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]:
```yaml
context: ci/openshift-jenkins/conformance
```

File: cluster/ci/config/prow/config.yaml [hunk 3]
```diff
@@ -269,14 +269,50 @@ presubmits:
...
+    rerun_command: "/test conformance"
+    trigger: "((?m)^/test( all| conformance),?(\\s+|$))"
+
+  - name: test_pull_request_ose_integration_and_end_to_end
+    agent: jenkins
+    labels:
+      master: ci.dev.openshift.redhat.com
+    always_run: true
+    context: ci/openshift-jenkins/ose
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]:
```yaml
context: ci/openshift-jenkins/integration
```

File: cluster/ci/config/prow/config.yaml [hunk 4]
```diff
@@ -269,14 +269,50 @@ presubmits:
...
+    rerun_command: "/test networking"
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]:
```yaml
rerun_command: "/test integration"
```

File: cluster/ci/config/prow/config.yaml [hunk 5]
```diff
@@ -269,14 +269,50 @@ presubmits:
...
+    trigger: "((?m)^/test( all| integration_and_end_to_end),?(\\s+|$))"
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]:
```yaml
trigger: "((?m)^/test( all| integration),?(\\s+|$))"
```

File: cluster/ci/config/prow/config.yaml [hunk 6]
```diff
@@ -269,14 +269,50 @@ presubmits:
...
+
+  - name: test_pull_request_ose_networking
+    agent: jenkins
+    labels:
+      master: ci.dev.openshift.redhat.com
+    always_run: true
+    context: ci/openshift-jenkins/ose
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]:
```yaml
context: ci/openshift-jenkins/networking
```

File: cluster/ci/config/prow/config.yaml [hunk 7]
```diff
@@ -269,14 +269,50 @@ presubmits:
...
+    rerun_command: "/test networking"
+    trigger: "((?m)^/test( all| networking),?(\\s+|$))"
+
+  - name: test_pull_request_ose_verify_commits
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: This job should not exist -- fold the single task in it into the check job. This takes like 10s and it doesn't make sense to spin up an entire VM for it

**Comment by paradigm** [comment]
[test]

**Pull request merged**